.github/actions/pre-commit-default/action.yml

```diff
@@ -0,0 +1,50 @@
+name: "Pre-commit format"
+description: "Install and run pre-commit"
+inputs:
+  pre-commit-args:
+    description: Additionnal parameters to pass to pre-commit
+    required: false
+  pre-commit-all-files:
+    description: >-
+      Whether to run pre-commit checks on all files (defaults to "true")
+    required: false
+    default: "true"
```

Comment:
I don't know if we really have an use case for when we don't want --all-files when running on CI

you got this by forking the existing pre-commit action right? maybe drop support for it until we figure out if it's really needed

Reply:
I thought about it and then kept it because it still allows me to handle formatting only on changed files from the caller using something like:

```
      - uses: Alfresco/alfresco-build-tools/.github/actions/pre-commit-default@AAE-11403-experiment-prettier-format
        with:
          pre-commit-all-files: false
          pre-commit-args: --files ${{ steps.file_changes.outputs.files }}
```

Now I'm ok removing all this as diff support would be included inside the action (if needed later), wdyt?

Reply:
(pushed: removed the input for all files)